lib/12T/ngspice/gf180mcu_osu_sc_12T_and2_1.spice

```diff
@@ -0,0 +1,11 @@
+* NGSPICE file created from gf180mcu_osu_sc_12T_and2_1.ext - technology: gf180mcuC
```

Comment:
this (and other `.spice` files) seems to still reference absolute hostname/username?

Reply:
hmm....okay..fixed.